### ISSUE TITLE
Jittery mouse

### ISSUE THREAD

**Issue by sambow23** [gamax92/wine-overwatch#33]
So I just installed Overwatch through Lutris, everything works fine except the fact that the mouse jitters when looking around. Any suggestions?

**Comment by IngeniousDox** [comment]
Some suggestions:

- Make sure your resolution and aspect ratio are set correctly ingame.

- Make sure your mouse settings in your desktop are set correctly outside of the game. Esp check various sources of mouse acceleration (https://wiki.archlinux.org/index.php/Mouse_acceleration).

**Comment by sambow23** [comment]
Nope, still broken... But oddly enough, when setting the Blizzard App to Windows XP, the mouse is fine. The problem about that is, Blizzard is removing XP support in later in September/October.

**Comment by IngeniousDox** [comment]
Check out the wiki: https://github.com/gamax92/wine-overwatch/wiki and read the install instructions.

You need to set battle.net to Win 7, and OW to WinXP. I think most of us have that because of the freezing issues, perhaps your jitter is a remnant of that.

**Comment by sambow23** [comment]
Yeah it is, but why does setting battle.net to windows 7 cause the mouse issue?
